packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -137,13 +137,22 @@ describe('ReactDOMFloat', () => {
     buffer = '';
   }
 
-  function getVisibleChildren(element) {
+  function getVisibleChildrenAndResources(element) {
```

Comment:
Kind of annoyingly long. `getVisibleChildren` or something like `getMeaningfulChildren` is fine.